### ISSUE TITLE
Failed: TOKEN_REVOKED

### ISSUE THREAD

**Issue by warmfusion** [solarce/signup.hangops.com#3]
When trying to use the signup page, i get the following error

> hangops.slack.com
> Failed! token_revoked

Does this service need to be reconfigured?

**Comment by sntxrr** [comment]
Hi, thanks for bringing this up - we've recently had some issues with some nefarious types joining and posting NSFW content.  We have purposefully disabled this while we bring up a replacement. Feel free to send me a DM with your email address (they are open), and I'll get you added. [https://twitter.com/sntxrrtoo](https://twitter.com/sntxrrtoo) - appreciate your issue report!

**Comment by Nklya** [comment]
@sntxrr there might be a typo, https://twitter.com/sntxrrtoo doesn't exist.
Maybe https://twitter.com/sntxrr ?

**Comment by sntxrr** [comment]
You are correct @Nklya - I recently updated my Twitter account to https://twitter.com/sntxrr - feel free to DM me there, DMs are open.
